### DISCUSSION TITLE
[Basketball] Having trouble finding specific stats for player positions and lineup slots

### DISCUSSION THREAD

**Discussion by zirekyle** [cwendt94/espn-api#545] [ESPN API] [answered]
So I'm actually working with WNBA data, which I know is in development, but seems to have mostly the same code as NBA. My overall goal is to list my team's lineup and highlight players that are currently in my lineup that are injured or don't have games today.

I have been able to accomplish this in a general sense, pulling player injury data and pulling the full schedule with _get_all_pro_schedule() and comparing pro teams to the schedule to see if today is present in the game list. This all works fine.

The problem arises with specific slots. I would like to know who is in my current lineup (meaning not in bench or IR slots) and which position each player is. I checked through teams, matchups, and box scores, and am not seeing a way to see who is in an active slot, just a general list of my players.

Additionally, position doesn't match what is on the ESPN GUI and eligible slots list all slots for every player. Is this because it is WNBA and not built out yet?

For example:

A'ja Wilson is a Center (on ESPN's main site), but listed as an SG in the API for position. Slot shows F, which is fine for F/C.
Courtney Vandersloot is on my bench but is listed as a lineup slot F as well, though she is a Guard. 
Jackie Young is in lineup slot G and is position Guard (both correct), but it doesn't show the lineup slot as UTIL.

Is there an accurate way to get the correct data here, or is this not present for WNBA?

**Comment by cwendt94** [comment, marked as answer]
Hey @zirekyle, you can get the slot position using box_scores and then accessing the teams lineup. Accessing would look something like this `box[0].away_lineup[0].slot_position` assuming `box` was the retuned valued from `box_scores`.

For the players position and lineup I am seeing the same thing where they are not correctly matching. Those values are set in this [constants](https://github.com/cwendt94/espn-api/blob/master/espn_api/wbasketball/constant.py#L1) file and seems like they were just copied over from the NBA constants. The hard part of getting the correct values is having to look through ESPN's API calls and manually match those numbers to the actual positions.

I have a [PR Here](https://github.com/cwendt94/espn-api/pull/552) which I think has most of the positions, I still can't figure out what position 8 and 9 are.

**Reply by zirekyle** [reply]
Thanks, subscribed to the PR.
